insights/specs/default.py

```diff
@@ -244,15 +244,16 @@ class DefaultSpecs(Specs):
     httpd24_httpd_error_log = simple_file("/opt/rh/httpd24/root/etc/httpd/logs/error_log")
     httpd_M = foreach_execute(httpd.httpd_cmds, "%s -M")
     httpd_V = foreach_execute(httpd.httpd_cmds, "%s -V")
+    httpd_conf = foreach_collect(httpd.httpd_configuration_files, "%s")
     httpd_cert_info_in_nss = foreach_execute(ssl_certificate.httpd_certificate_info_in_nss, '/usr/bin/certutil -d %s -L -n %s')
-    httpd_conf = glob_file(
-        [
-            "/etc/httpd/conf/httpd.conf",
-            "/etc/httpd/conf.d/*.conf",
-            "/etc/httpd/conf.d/*/*.conf",
-            "/etc/httpd/conf.modules.d/*.conf"
-        ]
-    )
+    # httpd_conf = glob_file(
+    #     [
+    #         "/etc/httpd/conf/httpd.conf",
+    #         "/etc/httpd/conf.d/*.conf",
+    #         "/etc/httpd/conf.d/*/*.conf",
+    #         "/etc/httpd/conf.modules.d/*.conf"
+    #     ]
+    # )
     httpd_conf_scl_httpd24 = glob_file(
```

Comment:
Better update the `httpd_conf_scl_httpd24` and `httpd_conf_scl_jbcs_httpd24` as well.